### PR TITLE
As2 37 put positive update a list

### PR DESCRIPTION
**Pull request by KolyaSamandasiuk**

### CODE REVIEW

File: src/test/java/api/clients/ListTestRestClient.java [hunk 1]
```diff
@@ -63,6 +62,19 @@ public ListsDataResponse moveList(String listId, String boardId) {
                 .as(ListsDataResponse.class);
     }
 
+    @Step("Update list information with id: {idList}")
+    public ListsDataResponse updateListFromBoard(Map<String, String> listKeyValue, String idList){
```
Comment (Goldbe54) [inline src/test/java/api/clients/ListTestRestClient.java]: add info about parameters(listKeyValue)

File: src/test/java/api/clients/ListTestRestClient.java [hunk 2]
```diff
@@ -39,7 +39,6 @@ public Response createList(Map<String, String> listKeyValue, String boardId, int
                 .response();
     }
 
-    @Step("Get lists by board id: {boardId}")
```
Comment (Goldbe54) [inline src/test/java/api/clients/ListTestRestClient.java:42]: bring back the step

File: src/test/java/api/put/UpdateListTest.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import static api.clients.ListTestRestClient.constructPutListKeyValue;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class UpdateListTest extends BaseTest {
+
+    private String idBoard;
+    private String idList;
+
+    @BeforeMethod
+    @Description("Create board and list")
```
Comment (Goldbe54) [inline src/test/java/api/put/UpdateListTest.java]: Use "Step" instead of "Description"

**Review by Goldbe54 — APPROVED** [review]

File: src/test/java/api/put/UpdateListTest.java [hunk 4]
```diff
@@ -20,7 +21,7 @@ public class UpdateListTest extends BaseTest {
     private String idList;
 
     @BeforeMethod
-    @Description("Create board and list")
+    @Step("Create board and list")
     public void createBoardAndList() {
         idBoard = boardRestTestClient.createNewBoard(constructDefaultBoardKeyValue()).getId();
         idList = listTestRestClient.createList(constructDefaultListKeyValue(), idBoard).getId();
```
Comment (KhraschukZahar) [inline src/test/java/api/put/UpdateListTest.java]: make it look like this 

@BeforeMethod
@Step("Test preparation")
 public void preconditions() {
       idBoard = boardRestTestClient.createNewBoard(constructDefaultBoardKeyValue()).getId();
       idList = listTestRestClient.createList(constructDefaultListKeyValue(), idBoard).getId();
    }

**Review by KhraschukZahar — APPROVED** [review]

**Pull request merged**